src/main/controllers/SubscriptionManagementController.ts

```diff
@@ -6,34 +6,40 @@ import {cloneDeep} from 'lodash';
 const subscriptionService = new SubscriptionService();
 
 export default class SubscriptionManagementController {
-  public get(req: PipRequest, res: Response): void {
+
+  public async get(req: PipRequest, res: Response): Promise<void> {
     if (req.user) {
       // currently only 2 users are mocked, userId: 1 has subscriptions, userId: 2 doesnt
       const userId = req.user['id'] === '1' ? 1 : 2;
-      const caseTableData = subscriptionService.generateCaseTableRows(userId);
-      const courtTableData = subscriptionService.generateCourtTableRows(userId);
-      let activeAllTab = false, activeCaseTab = false, activeCourtTab = false;
-      switch (Object.keys(req.query)[0]) {
-        case 'all':
-          activeAllTab = true;
-          break;
-        case 'case':
-          activeCaseTab = true;
-          break;
-        case 'court':
-          activeCourtTab = true;
-          break;
-        default:
-          activeAllTab = true;
-          break;
+      const tableData = await subscriptionService.generateSubscriptionsTableRows(userId);
```

Comment:
We can directly pass in req.user['id'] into this (and can remove line 13), so that the user ID is the one that the user has entered in mock-session.

Reply:
yes I agree. I changed it